frontend/model/contracts/group.js

```diff
@@ -1275,6 +1307,24 @@ sbp('chelonia/defineContract', {
         await sbp('gi.db/archive/save', archPaymentsKey, archPayments)
       }
       sbp('okTurtles.events/emit', PAYMENTS_ARCHIVED, { paymentsByPeriod, payments })
+    },
+    'gi.contracts/group/sendMincomeChangedNotification': async function (contractID, data) {
+      if (data.memberType === 'receiving') {
+        await sbp('gi.actions/group/groupProfileUpdate', {
```

Comment:
@taoeffect 
This part of the code is the source of below issue which I had trouble finding the cause for.

<img src='https://user-images.githubusercontent.com/17641213/211121117-45f0447e-8ffd-4bfa-89cf-fa9386ade359.png' width='410'>

Steps to reproduce / encounter this issue is:
1) Sign up `user-1` and then create a `group-1`(whose mincome is set to, let's say, 150$). then set up his income details as a pledger.
2) Sign up `user-2` and then join  him `group-2`. then set up his income details as a receiver with the income as let's say, 120$.
3) switch to `user-1` and reset the mincome to an amount lower than `user-2`'s income (e.g. 110$)
4) In `user-2`'s screen, we get this error.